pkg/common/policy/policy_examples.go

```diff
@@ -0,0 +1,349 @@
+package policy
+
+const defaultPermissionData = `
+{
+    "apis": [
```

Comment:
These defaults are now listed in three spots (MD, conf/server/permissions.json, and here). I think we should try and consolidate them into one spot. One solution is to create the `pkg/server/policy` package, include the default files in that location, and embed them into the binary using the `go:embed` directive (https://pkg.go.dev/embed). The markdown could then reference that file directly and we only have one spot to update things.

As I mentioned elsewhere, I don't think the "developer" conf under conf/ should configure the policy engine since most users will probably use the default.

Reply:
I am all for using embed.. is there any concern with the toolchain, i think `embed` is a fairly new golang feature - just checking if this is an issue for broader use cases or CI

>As I mentioned elsewhere, I don't think the "developer" conf under conf/ should configure the policy engine since most users will probably use the default.

yea agree - can change this to not set it and let it use the default.